backend/model/template/src/main/java/co/airy/model/template/TemplatePayload.java

```diff
@@ -0,0 +1,37 @@
+package co.airy.model.template;
+
+import co.airy.avro.communication.Template;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class TemplatePayload {
+    private String id;
+    private String name;
+    private String content;
+    private JsonNode variables;
+    public static TemplatePayload fromTemplate(final Template template) {
+        final ObjectMapper mapper = new ObjectMapper();
```

Comment:
Maybe we can use the objectmapper we. already have

Reply:
Hmm, I wouldn't. The templates are source specific message content. So we want to pass them as unmodified as possible, right?

Reply:
Good point 👍